modules/git/ref.go

```diff
@@ -24,17 +28,17 @@ func (ref *Reference) ShortName() string {
 	if ref == nil {
 		return ""
 	}
-	if strings.HasPrefix(ref.Name, "refs/heads/") {
-		return ref.Name[11:]
+	if strings.HasPrefix(ref.Name, BranchPrefix) {
+		return ref.Name[len(BranchPrefix):]
 	}
-	if strings.HasPrefix(ref.Name, "refs/tags/") {
-		return ref.Name[10:]
+	if strings.HasPrefix(ref.Name, TagPrefix) {
+		return ref.Name[len(TagPrefix):]
 	}
-	if strings.HasPrefix(ref.Name, "refs/remotes/") {
-		return ref.Name[13:]
+	if strings.HasPrefix(ref.Name, RemotePrefix) {
+		return ref.Name[len(RemotePrefix):]
 	}
-	if strings.HasPrefix(ref.Name, "refs/pull/") && strings.IndexByte(ref.Name[10:], '/') > -1 {
-		return ref.Name[10 : strings.IndexByte(ref.Name[10:], '/')+10]
+	if strings.HasPrefix(ref.Name, PullPrefix) && strings.IndexByte(ref.Name[pullLen:], '/') > -1 {
```

Comment:
Bit unfair that only PullPrefix gets a `xxxLen` constant ;)

Reply:
Yeah, I felt this would be overhead for the others. But consistency is important too.